apps/prairielearn/src/ee/lib/validateHTML.ts

```diff
@@ -560,11 +552,7 @@ function checkCheckbox(ast: DocumentFragment | ChildNode): ValidationResult {
 
         case 'partial-credit':
           assertBool('pl-checkbox', key, val, errors);
-          if (
-            ['false', 'f', '0', 'False', 'F', 'FALSE', 'no', 'n', 'No', 'N', 'NO'].includes(val)
-          ) {
-            usedPartialCredit = false;
-          }
+          usedPartialCredit = isBooleanTrue(val);
```

Comment:
Small drive-by fix: this attribute default to `false`, to the logic here and the default above were changed to reflect that.